packages/roosterjs-editor-core/lib/corePlugins/ImageSelection.ts

```diff
@@ -62,6 +62,17 @@ export default class ImageSelection implements EditorPlugin {
                         } else if (event.rawEvent.button === mouseLeftButton) {
                             this.editor.select(target);
                         }
+                    } else if (safeInstanceOf(target, 'HTMLElement')) {
+                        const mouseSelection = this.editor.getSelectionRangeEx();
```

Comment:
Can we just let the browser handle the mouse selection?

I think you can check this in MouseDown event instead, and if target element is not current image, just unselect it, then browser should be able to handle the rest.

Reply:
I understand. I also think we need to check if the last selection is a image to not cause problems with table cell selection.